common/test/acceptance/fixtures/__init__.py

```diff
@@ -1,11 +1,15 @@
 import os
 
 
+HOSTNAME = os.environ.get('BOK_CHOY_HOSTNAME', 'localhost')
+CMS_PORT = os.environ.get('BOK_CHOY_CMS_PORT', '8031')
```

Comment:
But this isn't a settings file, and hence doesn't inherit the settings loaded from the json files.  It seems like the bok-choy test fixtures actively avoided using the settings files, to the point of basically redefining parts of them.